src/installation/liveimages/baserequirements.md

```diff
@@ -0,0 +1,11 @@
+# Base Image System Requirements
+
+These are the minimum system requirements when using a live base image (without a bundled Desktop Environment)
```

Comment:
Lines that aren't part of some kind of syntactic structure (e.g. a table) should be wrapped at 80 columns.